src/BuiltInTools/dotnet-watch/HotReloadDotNetWatcher.cs

```diff
@@ -200,6 +206,12 @@ private static void ConfigureExecutable(DotNetWatchContext context, ProcessSpec
                 processSpec.EnvironmentVariables["ASPNETCORE_URLS"] = context.DefaultLaunchSettingsProfile.ApplicationUrl;
             }
 
+            var rootVariableName = Environment.Is64BitProcess ? "DOTNET_ROOT" : "DOTNET_ROOT(x86)";
```

Comment:
Is that really the name of the x86 root? That feels different that from what I remember.

Reply:
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet#environment-variables